cmake/sysbuild/suit.cmake

```diff
@@ -242,6 +261,13 @@ function(suit_setup_merge)
   sysbuild_get(BINARY_DIR IMAGE ${DEFAULT_IMAGE} VAR APPLICATION_BINARY_DIR CACHE)
   foreach(image ${IMAGES})
     set(ARTIFACTS_TO_MERGE)
+
+    sysbuild_get(GENERATE_LOCAL_ENVELOPE IMAGE ${image} VAR CONFIG_SUIT_LOCAL_ENVELOPE_GENERATE KCONFIG)
```

Comment:
This snippet is copied twice, maybe it is worth moving this into separate function?

Reply:
It would have to be a macro, because `continue` statement is used in the snippet. I think such change would make the script less clear.
Ideally, the `sysbuild_get` would unset a variable if it were not set, but that's not the case currently. I don't know why `sysbuild_get` works that way, but I wouldn't be surprised if it were just some detail causing problems.